### ISSUE TITLE
Slow IE11 performance loading items

### ISSUE THREAD

**Issue by asyncLiz** [vaadin/vaadin-grid#1479]
### Description

Settings items with an array equal to or larger than the physical row limit causes a noticeable slow performance in IE11. Profiling shows around 8 seconds to load the items, even though it is only rendering 25 physical rows.

### Expected outcome

Rendering time should not be noticeable (> 500ms) for a small table with 7 columns and 25 simple rows.

### Actual outcome

Grid takes increasingly longer as the array gets bigger.
![screen shot 2018-10-25 at 3 54 49 pm](https://user-images.githubusercontent.com/34382620/47530725-04e7bc00-d871-11e8-9b94-783dd11b82fa.png)
### Live Demo

https://github.com/asyncLiz/vaadin-grid-perf

### Steps to reproduce

1. Clone the above repo
2. `npm install`
3. `npm start`
4. Open index.html in IE and start profiler before clicking on one of the load buttons

### Browsers Affected

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [X] IE 11
- [ ] iOS Safari
- [ ] Android Chrome

**Comment by asyncLiz** [comment]
@tomivirkki I have noticed that it gets faster after the initial items are set. Is it still just as fast when you click on "Load 500 Items" first after the page loads? If I work up from 1 -> 5 -> 25 -> 500 it's much smoother.

The profile above was done on BrowserStack, but I repeated the test on a high end Windows machine last night. It was faster, but still a noticeable delay at around 3 seconds to load 500 items immediately after the page loads in.

In my profiling, I saw that the `textContent` of all the cells was being set twice since https://github.com/vaadin/vaadin-grid/blob/master/src/vaadin-grid-column.html#L369 does not check if the element needs updating. Re-setting the `textContent` caused unneeded style updates.

Replacing the default renderer with
```
const pathRenderer = (root, owner, {item}) => {
  const text = this.get(path, item);
  if (root.textContent !== text) {
    root.textContent = this.get(path, item);
  }
}
```

resulted in a net gain of around 300ms.
